File: src/main/java/com/example/letmovie/domain/reservation/dto/response/ShowTimeResponseDTO.java
package com.example.letmovie.domain.reservation.dto.response;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class ShowTimeResponseDTO {
    private String theaterName;       // 극장이름
    private String screenName;        // 상영관 이름
    private String screenTotalSeat;   // 상영관 전체 좌석
    private String screenRemainSeat;  // 상영관 예약 가능 좌석
    private String showtime;          // 상영 시작 시간
    private String showtimeId;        // 쇼타임 ID
}

File: src/main/java/com/example/letmovie/domain/reservation/dto/response/ShowTimeResponseDTO.kt
package com.example.letmovie.domain.reservation.dto.response

class ShowTimeResponseDTO (
    var theaterName: String? = null, // 극장이름
    var screenName: String? = null,// 상영관 이름
    var screenTotalSeat: Int = 0, // 상영관 전체 좌석
    var screenRemainSeat: Int = 0, // 상영관 예약 가능 좌석
    var showtime: String? = null, // 상영 시작 시간
    var showtimeId: Long? = null // 쇼타임 ID
)